testsuite/run_sets/testsuite.yml

```diff
@@ -22,18 +22,18 @@
   # one of: core_proxy_register_as_trad_with_script.feature
   #         core_proxy_register_as_minion_with_script.feature
   #         core_proxy_register_as_minion_with_gui.feature
-- features/core_proxy_register_as_minion_with_gui.feature
-- features/core_proxy_branch_network.feature
+#- features/core_proxy_register_as_minion_with_gui.feature
```

Comment:
No. This looks like an improper merge conflict resolution. Please revert as it was before.

Reply:
No. I commented it in ubuntu-support branch.
We don't need proxy, and that will make pass our tests faster, until we have a stable testsuite, then we enable again all these features ;)